src/docs/common/snippets/common-install-graalvm-sdkman.adoc

```diff
@@ -14,3 +14,22 @@ sdk use java 17.0.8-graal
 
 For installation on Windows, or for manual installation on Linux or Mac, see the https://www.graalvm.org/latest/docs/getting-started/[GraalVM Getting Started] documentation.
 
+
+The previous command installs Oracle GraalVM, which is free to use in production and free to redistribute, at no cost, under the
+> [GraalVM Free Terms and Conditions](https://www.oracle.com/downloads/licenses/graal-free-license.html).
+
+Alternatively, you can use https://github.com/graalvm/graalvm-ce-builds/releases/[GraalVM Community Edition]:
```

Comment:
Why would I do this? If the other one is free to use in production, why would I want to use the community edition?

Maybe it's just a gap in my knowledge